http/digest/src/test/java/org/wildfly/security/http/digest/DigestMechanismFactoryTest.java

```diff
@@ -0,0 +1,83 @@
+package org.wildfly.security.http.digest;
```

Comment:
@keshav-redhat just a minor, please add the copyright message on top of each new class. Example is [here](https://github.com/wildfly-security/wildfly-elytron/blob/1.x/http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcBaseTest.java#L1)